### DISCUSSION TITLE
Hyas Doks theme or Doks project?

### DISCUSSION THREAD

**Discussion by pensivedog** [gethyas/hyas#372] [General]
Is the preferred way to install and use Doks to install Hyas and select Doks as the theme, as per Hyas installation instructions? Or should we follow the installation instructions on getdoks.org?

**Comment by h-enk** [comment]
> Is the preferred way to install and use Doks to install Hyas and select Doks as the theme, as per Hyas installation instructions?

Just not yet (working hard on getting there).

> Or should we follow the installation instructions on getdoks.org?

Until then, yes.

**Reply by pensivedog** [reply]
Thanks for the reply. Installing Hyas and selecting Doks as the theme seems to work fine right now. Any reason not to go with this approach?

**Reply by h-enk** [reply]
Because it not yet uses the new structure (working on it) and will fail in production

**Reply by pensivedog** [reply]
Just tried using the `npm create hyas@latest` and selecting Doks I get the following build error:

```
Error: error building site: render: failed to render pages: render of "page" failed: "...path...\doks\node_modules\@hyas\core\layouts\_default\baseof.html:3:5": execute of template failed: template: _default/single.html:3:5: executing "_default/single.html" at <partial "head/head" .>: error calling partial: partial "head/head" timed out after 30s. This is most likely due to infinite recursion. If this is just a slow template, you can try to increase the 'timeout' config setting.
```

**Reply by h-enk** [reply]
Did you try rasing the [timeout](https://gohugo.io/getting-started/configuration/#timeout) value? Try e.g. with `timeout = "1m"` in `./config/_default/hugo.toml`.

> You might need to raise this limit if your pages are slow to generate (e.g., because they require large image processing or depend on remote contents).

Note that, the initial `npm run dev` takes notably longer than subsequent runs. The latter make (smartly) use of  cached resources,

**Reply by pensivedog** [reply]
Ah, my apologies for the false report! I just tried again without setting the timeout and it worked. Thank you!

**Reply by h-enk** [reply]
No worries! Thanks for bringing this up because it helps to improve Hyas/Doks.
